helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java

```diff
@@ -546,8 +546,15 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
       builder.setToken(token);
     }
 
-    builder.setAddCapabilities(options.<String>getList(addCapabilityArg.getDest()));
-    builder.setDropCapabilities(options.<String>getList(dropCapabilityArg.getDest()));
+    final List<String> addCaps = options.<String>getList(addCapabilityArg.getDest());
```

Comment:
I don't have a strong preference, just trying to fix a broken part of the current implementation